.github/workflows/templates/generate.py

```diff
@@ -229,20 +240,25 @@
     "centos-stream9",
     "debian-10",
     "debian-11",
-    "fedora-35",
     "fedora-36",
     "fedora-37",
     "fedora-38",
     "gentoo",
     "gentoo-systemd",
     "opensuse-15",
     "opensuse-tumbleweed",
+    "photon-3",
+    "photon-4",
     "rockylinux-9",
     "ubuntu-2004",
     "ubuntu-2110",
     "ubuntu-2204",
 ]
 
+BLACKLIST_GIT_MASTER = [
+    "photon-4",
```

Comment:
We can't install from git master on PhotonOS 4?

Reply:
@s0undt3ch There was a traceback during the install about missing libraries, I wanted to get the initial support merged in and then investigate that later.